core/src/main/java/com/datastax/oss/driver/internal/core/metadata/token/SimpleReplicationStrategy.java

```diff
@@ -63,9 +67,9 @@ private static Token getTokenWrapping(int i, List<Token> ring) {
     return ring.get(i % ring.size());
   }
 
-  private static int extractReplicationFactor(Map<String, String> replicationConfig) {
+  private static ReplicationFactor extractReplicationFactor(Map<String, String> replicationConfig) {
     String factorString = replicationConfig.get("replication_factor");
     Preconditions.checkNotNull(factorString, "Missing replication factor in " + replicationConfig);
-    return Integer.parseInt(factorString);
+    return ReplicationFactor.fromString(factorString);
```

Comment:
to sum up this change, the transient replicas are only loaded and parsed, but currently, they are ignored by the `ReplicationStrategy`, right?

Reply:
Yes. They won't be used for token-aware routing, nor surfaced in token metadata. I think that's the right thing to do (see my other comment).

Reply:
That is correct see documentation here for more information on how this is working under the covers.

http://cassandra.apache.org/blog/2018/12/03/introducing-transient-replication.html